### PR TITLE
how to build with polkadot in Chinese

### PR DESCRIPTION
**Pull request by ansonla3**

### CODE REVIEW

File: docs/lang/chinese.md [hunk 1]
```diff
@@ -10,6 +10,12 @@
 
 [Governance - 治理](https://mp.weixin.qq.com/s/yyPaHsU7Nf7sUBVGIq1jbw)
 
+
+## Prachain 平行链资料
```
Comment (lsaether) [inline docs/lang/chinese.md]:
```suggestion
## Parachain 平行链资料
```

File: docs/lang/build/build-with-polkadot.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
+# 如何在Polkadot上开始建立
+
+Polkadot在[白皮书](https://github.com/w3f/polkadot-white-paper/raw/master/PolkaDotPaper.pdf)中被描述为完全可扩展和可扩容的区块链开发，部署和交互测试平台。开发者能够将这种令人兴奋的新技术用作其自身应用程序的基础架构。
+
+本指南将引导你了解Polkadot目前的开发状况，解释平行链和智能合约之间的区别，逐步带你走过需要的步骤，从开始一个新项目建立平行链到成为Polkadot上的链或者从选择平台到支付部署来建立智能合约。
+
+## 我们处于哪里？
+
+Polkadot已经实现了PoC-3的里程碑，并成功运行了Alexander 测试网络三个月。在下一个PoC-4版本中将逐步取得进展，其中包括跨链消息传递。开发进度有望在今年年底前推出主网。
```
Comment (xlc) [inline docs/lang/build/build-with-polkadot.md]: PoC 4不是已经release了么

File: docs/lang/build/build-with-polkadot.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+与此同时，Polkadot是利用Substrate的runtime模块库来构建的，它也刚刚发布了第一个稳定版本。 Substrate允许开发者重复使用许多用于创建Polkadot时的模块将它们组合成自己的链。通过编写定制链runtime的逻辑来自定义runtime模块也可以轻松扩展。
```
Comment (xlc) [inline docs/lang/build/build-with-polkadot.md]:
```suggestion
与此同时，Polkadot是利用Substrate的runtime模块库来构建的，它也刚刚发布了第一个稳定版本。 Substrate允许开发者重复使用许多用于创建Polkadot时的模块，将它们组合成自己的链。通过编写定制链runtime的逻辑来自定义runtime，模块也可以轻松扩展。
```
最后一句不太懂什么意思

File: docs/lang/build/build-with-polkadot.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+如果你是开发者、建设者、企业家或其他有远见的人，将Polkadot视为你下一个项目的生态系统，本指南将帮助你了解开发的选项，并确定哪种堆栈适合你和你的项目，并会向你介绍现在可开始计划和执行你想法的资源。
+
+## 平行链与智能合约有什么区别?
+
+在你开始实行你的想法之前，你需要了解在Polkadot上构建的两个开发选项之间的权衡：平行链或智能合约。
+
+平行链是独立的链连接到Polkadot网络取得Polkadot中继链验证人上的共享安全。Polkadot本身正在使用相同的工具来开始创建平行链。平行链允许灵活性和对链逻辑的限制较少，但是带来一些额外的处理，这使得它与智能合约相比并不总是更好的选择。
```
Comment (xlc) [inline docs/lang/build/build-with-polkadot.md]:
```suggestion
平行链是独立的链，连接到Polkadot网络取得Polkadot中继链验证人上的共享安全。Polkadot本身正在使用相同的工具来开始创建平行链。平行链允许灵活性和对链逻辑的限制较少，但是需要一些额外的处理，这使得它与智能合约相比各有千秋。
```

File: docs/lang/build/build-with-polkadot.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+对Polkadot的智能合约来说，现在仍然还是很早而且开发才刚刚稳定下来。我们积极地提供内容去帮助开发者加快速度和维持wiki最新的资料。您还应该通过以下链接获取最新消息：
+
+
+### Edgeware
+
+Edgeware的计划了在Polkadot上运行平行链，它将会是个智能合约平台。随着有趣的治理和代币分配创新，它将会是第一个连接到在Polkadot主网上带有智能合约的平行链。你可以在它们的网站上了解其项目的最新信息。
+
+### pDSL（Parity领域专用语言）
+[pDSL](https://github.com/Robbepop/pdsl)有意成为用于编写Rust智能合约的新领域专用语言，把Rust编译成Wasm代码。如README中所述，它仍处于实验阶段，缺少了很多计画中的功能，但今天是有可能开始使用它来编写智能合约。
```
Comment (xlc) [inline docs/lang/build/build-with-polkadot.md]: ink

**Review by xlc — COMMENTED** [review]
有些是繁体有些是简体？

**Comment by ansonla3** [comment]
> 有些是繁体有些是简体？

嗯。 所有也是初步先放上來 還有些沒開始 和 會重複再看呢。過2天。 再看吧 ：）

File: docs/lang/build/build-with-polkadot.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+
+Polkadot也会通过桥接(Bridge)方式把现有的智能合约平台(例如:以太坊 & Tezos)连接起来，这意味着即使今天在这些平台上开发的工作，日后也可能适用于在Polkadot上运行。
+
+### Edgeware
+
+Edgeware的计划了在Polkadot上运行平行链，它将会是个智能合约平台。带有具创新的治理和代币分配，它将会是第一个连接到在Polkadot网络上带有智能合约的平行链。你可以在它们的[网站](https://edgewa.re/)上了解其项目的最新消息。
+
+### Ink
+
+[Ink](https://github.com/paritytech/ink)有意成为用于编写Rust智能合约的新领域专用语言，把Rust编译成Wasm代码。如README中所述，它仍处于实验阶段，缺少了很多计画中的功能，但现在是可以开始使用它来编写智能合约。
```
Comment (xlc) [inline docs/lang/build/build-with-polkadot.md]:
```suggestion
[Ink](https://github.com/paritytech/ink)有意成为用于编写Rust智能合约的新领域专用语言，把Rust编译成Wasm代码。如README中所述，它仍处于实验阶段，缺少了很多计划中的功能，但现在是可以开始使用它来编写智能合约。
```

File: docs/lang/build/build-with-polkadot.md [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+
+对于感兴趣的开发者，你们可以通过学习经已编写好的[例子](https://github.com/Robbepop/pdsl/tree/master/examples)开始来编写智能合约。这些可以用来作为参考编写更加复杂的逻辑并将可以部署在支持智能合约的平行链上。
```
Comment (xlc) [inline docs/lang/build/build-with-polkadot.md]:
```suggestion
对于感兴趣的开发者，你们可以通过学习经已编写好的[例子](https://github.com/paritytech/ink/tree/master/examples)开始来编写智能合约。这些可以用来作为参考编写更加复杂的逻辑并将可以部署在支持智能合约的平行链上。
```

**Review by lsaether — COMMENTED** [review]
Why not make a `docs/lang/chinese/` folder and put everything in there instead of putting a build folder inside lang?

**Comment by ansonla3** [comment]
> Why not make a `docs/lang/chinese/` folder and put everything in there instead of putting a build folder inside lang?

that's good idea for the long run

**Comment by lsaether** [comment]
Can you double check that all relative links still work. IE, those that are like `../../polkadot/...` ?

**Review by lsaether — APPROVED** [review]

**Pull request merged**